packages/tasks/src/tasks/object-detection/spec/input.json

```diff
@@ -6,7 +6,8 @@
 	"type": "object",
 	"properties": {
 		"inputs": {
-			"description": "The input image data"
+			"type": "string",
+			"description": "The input image data as a base64-encoded string. If no `parameters` are provided, you can also provide the image data as a raw bytes payload."
```

Comment:
(not a change request)

The reason why we didn't type this was not to be biased towards the HTTP API (as client code might also support other ways, such a file descriptor and so on).

The generated typescript code does not reflect that you can pass a `Buffer` for example: in the future we might want to add a piece of code that detects such cases (binary data input) and generates the correct type / code

(that being said, I think the proposed change is better as we didn't even mention base64 / raw data in the previous version 😄 )

Reply:
Yes true. My perspective is to see these specs more as an API specification rather than a clients specification (i.e. rather than which inputs/outputs the clients should have). In Python we also handle many types as input (raw bytes, url, filepath, base64 string, etc.) but always send the data to the API using the correct format (raw bytes or base64). I would also like to have a better way to describe this but for now that's the best I can think of :confused: